src/NaiveBayes.jl

```diff
@@ -16,7 +16,10 @@ export NBModel,
         predict_proba,
         predict_logprobs,
         restructure_matrix,
-        to_matrix
+        to_matrix,
+        write_model,
+        load_model,
+        get_features_names
```

Comment:
I'm not a native speaker as well so I really wouldn't count me as an authority. I think you are right so I'll make the change